### ISSUE TITLE
[QUESTION] v0.6.5.0 stable. How to adjust CPU thread count now that xmrig isnt used?

### ISSUE THREAD

**Issue by bminei** [nicehash/NiceHashQuickMiner#951]
**Has the question been answered in [Wiki](https://github.com/nicehash/NiceHashQuickMiner/wiki)? There is no such question in [FAQ](https://github.com/nicehash/NiceHashQuickMiner/wiki/FAQ) or [Troubleshooting collection](https://github.com/nicehash/NiceHashQuickMiner/wiki/Troubleshooting)?**
No

**Your question is...**
How do I adjust CPU max threads now that randomx opens in excavator v1.8.5.0 and not xmrig?  Changing "--cpu-max-thread-hint 50" in "Edit config file" no longer works and just pegs my CPU usage to 100% (25 threads used out of 32 on AMD 3950x).

**Comment by Ezzassin** [comment]
Under NHQM file..."CPUMinerELP" : "NTHREADS=0,HIGHPRIORITY=0,USELARGEPAGE=1,USEMSR=1",
NT=number of threads...If you dont have that option you need to reset your NHQM file.

**Comment by bminei** [comment]
I was able to control number of threads by reinstalling, but I've ran into new problems.  If my computer has been on for a while, excavator runs into a boot loop when turning on cpu mining (goes away when cpu mining is turned off).  I've logged it, but no errors are shown.  Once I restart my computer the boot loop goes away, but errors occur.  I get an msr driver error, but if I close excavator and let it restart automatically, the errors go away.  Now randomx is able to mine and accept shares but after a minute all shares are over target and never recovers.  At the same time my computer becomes very sluggish with some apps not being able to open.  I've set it to use half my threads available and I don't have any hardware monitoring apps running.  I've reverted back to quickminer v0.6.1.2 stable and have no problems cpu mining.
